python/nano/src/bigdl/nano/tf/keras/customized_training_utils.py

```diff
@@ -132,6 +133,22 @@ def __call__(self, *args, **kwargs):
                     cloudpickle.dump(arg, f)
                     new_args.append(("others", os.path.join(temp_dir, f"args_{i}.pkl")))
 
+            # check necessary objects
+            arg_instance = list(map(lambda x: x[0], new_args))
+            invalidInputError("model" in arg_instance, "Please check whether 'model' in the input "
+                                                       "parameter in train function.")
```

Comment:
this is quite strange to understand for users since we do not require a parameter called `'model'`. We may just make this "Please check if a model inherited from nn.Module is inputted as parameter in train function."